src/System.IO.Ports/tests/Configurations.props

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard-Windows_NT;
-      netstandard;
       netfx;
     </BuildConfigurations>
```

Comment:
What we had originally was just `netstandard-Windows_NT`, which definitely *was* running netfx tests but (I think) wasn't running core tests.  I'm not sure removing netstandard will be enough to fix this.